.github/workflows/phpcs.yml

```diff
@@ -0,0 +1,53 @@
+name: PHPCS
+
+on:
+  pull_request:
+
+jobs:
+  coding-standards:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+
+    name: Coding Standards
+    steps:
+      - name: Cancel previous runs of this workflow (pull requests only)
+        if: ${{ github.event_name == 'pull_request' }}
+        uses: styfle/cancel-workflow-action@0.5.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Check out code
+        uses: actions/checkout@v2
```

Comment:
Another action suggestion:

```

- name: Check Gitignored files
        run: if [[ ! -z $(git ls-files -i --exclude-standard) ]]; then exit 1; fi
```

Reply:
Is this something that is likely to be an issue? What would a developer have to do?